modules/core/src/main/scala/Transaction.scala

```diff
@@ -141,11 +144,15 @@ object Transaction {
       s.execute(internal"ROLLBACK".command)
 
     def doCommit: F[Completion] =
-      s.execute(internal"COMMIT".command)
+      s.execute(internal"COMMIT".command)      
 
     val acquire: F[Transaction[F]] =
       assertIdle(CallSite("begin", Origin.unknown)) *>
-      s.execute(internal"BEGIN".command).map { _ =>
+      s.execute(
+        internal"""BEGIN
+                  ${iOp.map(i => s"ISOLATION LEVEL ${i.sql}").getOrElse("")}
```

Comment:
Can you change this to `iOp.foldMap(i => s"ISOLATION LEVEL ${i.sql}")` here and similarly below so you don't need the `getOrElse("")`?

Reply:
Good idea! :) Done!